hack/generate-latest-csv.sh

```diff
@@ -13,11 +13,11 @@ export CSV_CHECKSUM_OUTFILE="hack/latest-csv-checksum.md5"
 export CSV_VERSION=0.0.1
 
 # Current dependency images our DEV CSV are pinned to
-export ROOK_IMAGE=${ROOK_IMAGE:-"rook/ceph:v1.1.2"}
+export ROOK_IMAGE=${ROOK_IMAGE:-"rook/ceph:master"}
```

Comment:
this is a floating tag. can we use an exact release here?

Reply:
@davidvossel I requested to have the ocs-operator master branch to use rook master so that we catch problems with rook master. the ocs-4.2 release branch should use an exact release. 

We *could* use the (floating) tip of a rook release branch (release-1.1).

Reply:
@rohantmp pointed out to me that we currently don't have a floating build from the release-1.1 branch in docker hub: https://hub.docker.com/r/rook/ceph/tags - this exists only for master.

So I'm going to merge this now, and we can discuss whether to go to v1.1.4 for now, or wait until we have a floating build from the release-1.1 branch of rook and move to that.

Reply:
/lgtm